bitnami/cilium/values.yaml

```diff
@@ -0,0 +1,1515 @@
+  ##
+  podAnnotations: {}
+  ## @param operator.podAffinityPreset Pod affinity preset. Ignored if `operator.affinity` is set. Allowed values: `soft` or `hard`
+  ## ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity
+  ##
+  podAffinityPreset: ""
+  ## @param operator.podAntiAffinityPreset Pod anti-affinity preset. Ignored if `operator.affinity` is set. Allowed values: `soft` or `hard`
+  ## ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity
+  ##
+  podAntiAffinityPreset: soft
+  ## Node operator.affinity preset
+  ## ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity
+  ##
+  nodeAffinityPreset:
+    ## @param operator.nodeAffinityPreset.type Node affinity preset type. Ignored if `operator.affinity` is set. Allowed values: `soft` or `hard`
+    ##
+    type: ""
+    ## @param operator.nodeAffinityPreset.key Node label key to match. Ignored if `operator.affinity` is set
+    ##
+    key: ""
+    ## @param operator.nodeAffinityPreset.values Node label values to match. Ignored if `operator.affinity` is set
+    ## E.g.
+    ## values:
+    ##   - e2e-az1
+    ##   - e2e-az2
+    ##
+    values: []
+  ## @param operator.affinity Affinity for Cilium Operator pods assignment
+  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+  ## NOTE: `operator.podAffinityPreset`, `operator.podAntiAffinityPreset`, and `operator.nodeAffinityPreset` will be ignored when it's set
+  ##
+  affinity: {}
+  ## @param operator.nodeSelector [object] Node labels for Cilium Operator pods assignment
+  ## ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/
+  ##
+  nodeSelector:
+    kubernetes.io/os: linux
+  ## @param operator.tolerations Tolerations for Cilium Operator pods assignment
+  ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  ##
+  tolerations: []
+  ## @param operator.updateStrategy.type Cilium Operator deployment update strategy type
+  ## Can be set to RollingUpdate or Recreate
+  ## ref: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy
+  ##
+  updateStrategy:
+    type: RollingUpdate
+  ## @param operator.priorityClassName Cilium Operator pods' priorityClassName
+  ##
+  priorityClassName: ""
+  ## @param operator.topologySpreadConstraints Topology Spread Constraints for Cilium Operator pod assignment spread across your cluster among failure-domains
+  ## ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/#spread-constraints-for-pods
+  ##
+  topologySpreadConstraints: []
+  ## @param operator.schedulerName Name of the k8s scheduler (other than default) for Cilium Operator pods
+  ## ref: https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers/
+  ##
+  schedulerName: ""
```

Comment:
The schedulerName is not in the pod definition, in this case, as it is a deployment it makes sense

Reply:
done at https://github.com/bitnami/charts/pull/26374/commits/129afd439cbffb07203c84e7db2880f82caa370f